### ISSUE TITLE
CBLAuthDelegate::requestRefresh don't use correct timeout

### ISSUE THREAD

**Issue by l4468430** [alexa/avs-device-sdk#1106]
```
avsCommon::utils::libcurlUtils::HTTPResponse CBLAuthDelegate::requestRefresh() {
    .....
    auto timeout = m_configuration->getRequestTimeout();
    if (AuthObserverInterface::State::REFRESHED == m_authState) {
        auto timeUntilExpired = std::chrono::duration_cast<std::chrono::seconds>(m_tokenExpirationTime - m_requestTime);
        // https reqest timeout must be less than access token expire time
        if (timeout > timeUntilExpired) {
            timeout = timeUntilExpired;
        }
    }

    return m_httpPost->doPost(
        m_configuration->getRequestTokenUrl(), headerLines, postData, m_configuration->getRequestTimeout());//I think the parameter using m_configuration->getRequestTimeout should be timeout.
}
```

**Comment by celinval** [comment]
Great catch! Thanks! We'll update our code.

**Comment by scotthea-amazon** [comment]
This is fixed in v1.11.  Closing this issue.  If you think this issue has been closed in error, please re-open it and include a comment describing why it should not have been closed.
